extension/jaegerremotesampling/go.mod

```diff
@@ -12,70 +12,41 @@ require (
 
 require (
 	cloud.google.com/go/compute v1.5.0 // indirect
-	contrib.go.opencensus.io/exporter/prometheus v0.4.0 // indirect
```

Comment:
Yeah, ran `make gotidy` and was wondering if I should just leave it because it's just going to be hanging everytime that runs. but ok I'll remove them.

Reply:
Updated